### DISCUSSION TITLE
Local seqr superuser unable to edit

### DISCUSSION THREAD

**Discussion by jlieviant** [broadinstitute/seqr#3049] [Q&A] [answered]
We installed a local instance of seqr using docker compose. Most of the functions work, but even as root I am unable to edit Proband relationships (Error: Permission denied) or on the variant_search page, toggle or edit tags. I checked the permissions table and the user should have all of them. I'm not sure if there's something else I missed.

**Comment by hanars** [comment]
You need to log in in the seqr UI as a user that has been granted superuser permissions

**Comment by jlieviant** [comment]
I should clarify that I'm already logged in as a user with superuser permissions, but is still unable to make use of some functions.

**Comment by hanars** [comment, marked as answer]
Here is a good summary of the various permission levels available in seqr: https://github.com/broadinstitute/seqr/discussions/2853

Proband relationship is restricted to analyst users, so make sure you have that permission enabled for your user

**Reply by jlieviant** [reply]
I've solved the issue, thank you for the assistance!
